lib/src/home_page.dart

```diff
@@ -15,43 +15,101 @@ class _HomePageState extends State<HomePage> {
   @override
   Widget build(BuildContext context) {
     final localizations = AppLocalizations.of(context)!;
+    final theme = Theme.of(context);
+    final customColor = Theme.of(context).extension<CustomColors>()!;
     return Scaffold(
       appBar: AppBar(
-        title: Center(
-          child: Text(localizations.title),
+        backgroundColor: theme.appBarTheme.backgroundColor,
+        title: Text(
+          localizations.cards,
+          style: TextStyle(
+            color: theme.colorScheme.primary,
+            fontWeight: FontWeight.bold,
+          ),
         ),
+        actions: [
+          IconButton(
```

Comment:
Mete a ir logo para as settings. Não havendo um use case já explicito (não vejo a utilidade do help :p ) para is ser um menu assim, mais vale reduzir o # numero de clicks.